File: Assignment 11/Java/8.3/RoomCarpet.java
class RoomCarpet {
    private RoomDimension dimension;
    private double costPerSquareFoot;

    // Constructor
    public RoomCarpet(RoomDimension dimension, double costPerSquareFoot) {
        this.dimension = dimension;
        this.costPerSquareFoot = costPerSquareFoot;
    }

    public double getTotalCost() {
        return dimension.getArea() * costPerSquareFoot;
    }
}

File: Assignment 12/Tony Gradis C++/14.13/Code/RoomCarpet.cpp
#include "RoomCarpet.h"
#include <iostream>
#include <iomanip>

RoomCarpet::RoomCarpet(const RoomDimension& rd, double cost)
    : room(rd), costPerSquareFoot(cost) {}

double RoomCarpet::getTotalCost() const {
    return room.getArea() * costPerSquareFoot;
}
